bsoid_app.py

```diff
@@ -1,3 +1,18 @@
+import os
+
+working_dir = os.environ.get('WORKING_DIR_BSOID', '')
+prefix = os.environ.get('PREFIX_BSOID', '')
+framerate = os.environ.get('FRAMERATE_BSOID', '')
+software_choice = os.environ.get('SOFTWARE_BSOID', '')
```

Comment:
If you want to go above and beyond you could even do error handling, like if any of these are actually empty, it should probably error out and not default to `""`